client/my-sites/plugins/categories/use-categories.tsx

```diff
@@ -166,7 +166,7 @@ export const getCategories: () => Record< string, Category > = () => ( {
 	},
 	booking: {
 		menu: __( 'Booking & Scheduling' ),
-		title: __( 'Booking & Scheduling' ),
+		title: __( 'Booking & scheduling' ),
```

Comment:
I've also updated all these category headers to use sentence case.

The `menu` string is used for the filter options at the top. Are we ok with the filters continuing to use title case @fditrapani?

![Screenshot 2025-03-19 at 4 09 37 PM](https://github.com/user-attachments/assets/5a50d4b8-28f1-44d2-b8cc-4cb0415c091d)